diff -Nur a/libc.in b/libc.in
--- a/libc.in	2017-02-19 11:57:46.152075777 +0100
+++ b/libc.in	2017-02-19 11:59:21.540076800 +0100
@@ -95,7 +95,7 @@
 		resolv_conf_tail="$(cat "$SYSCONFDIR"/resolv.conf.d/tail)"
 	fi
 fi
-: ${resolv_conf:=/etc/resolv.conf}
+: ${resolv_conf:=/run/network/resolv.conf}
 : ${libc_service:=nscd}
 : ${list_resolv:=@SBINDIR@/resolvconf -l}
 if [ "${resolv_conf_head-x}" = x -a -f "$SYSCONFDIR"/resolv.conf.head ]; then
diff -Nur a/resolvconf.8.in b/resolvconf.8.in
--- a/resolvconf.8.in	2017-02-19 11:57:46.152075777 +0100
+++ b/resolvconf.8.in	2017-02-19 12:01:41.472078301 +0100
@@ -51,12 +51,12 @@
 .Xr resolv.conf 5
 files from multiple sources, such as DHCP and VPN clients.
 Traditionally, the host runs just one client and that updates
-.Pa /etc/resolv.conf .
+.Pa /run/network/resolv.conf .
 More modern systems frequently have wired and wireless interfaces and there is
 no guarantee both are on the same network.
 With the advent of VPN and other
 types of networking daemons, many things now contend for the contents of
-.Pa /etc/resolv.conf .
+.Pa /run/network/resolv.conf .
 .Pp
 .Nm
 solves this by letting the daemon send their
@@ -70,7 +70,7 @@
 instead of the filesystem.
 .Nm
 then updates
-.Pa /etc/resolv.conf
+.Pa /run/network/resolv.conf
 as it thinks best.
 When a local resolver other than libc is installed, such as
 .Xr dnsmasq 8
@@ -85,7 +85,7 @@
 In some situations
 .Nm
 needs to act as a deterrent to writing to
-.Pa /etc/resolv.conf .
+.Pa /run/network/resolv.conf .
 Where this file cannot be made immutable or you just need to toggle this
 behaviour,
 .Nm
@@ -275,7 +275,7 @@
 .El
 .Sh FILES
 .Bl -ohang
-.It Pa /etc/resolv.conf.bak
+.It Pa /run/network/resolv.conf.bak
 Backup file of the original resolv.conf.
 .It Pa @SYSCONFDIR@/resolvconf.conf
 Configuration file for
diff -Nur a/resolvconf.conf b/resolvconf.conf
--- a/resolvconf.conf	2017-02-19 11:57:46.152075777 +0100
+++ b/resolvconf.conf	2017-02-19 12:01:59.976078499 +0100
@@ -1,7 +1,7 @@
 # Configuration for resolvconf(8)
 # See resolvconf.conf(5) for details
 
-resolv_conf=/etc/resolv.conf
+resolv_conf=/run/network/resolv.conf
 # If you run a local name server, you should uncomment the below line and
 # configure your subscribers configuration files below.
-#name_servers=127.0.0.1
\ Kein Zeilenumbruch am Dateiende.
+#name_servers=127.0.0.1
diff -Nur a/resolvconf.conf.5.in b/resolvconf.conf.5.in
--- a/resolvconf.conf.5.in	2017-02-19 11:57:46.152075777 +0100
+++ b/resolvconf.conf.5.in	2017-02-19 11:59:48.740077092 +0100
@@ -145,7 +145,7 @@
 .Bl -tag -width indent
 .It Sy resolv_conf
 Defaults to
-.Pa /etc/resolv.conf
+.Pa /run/network/resolv.conf
 if not set.
 .It Sy resolv_conf_options
 A list of libc resolver options, as specified in
