diff -Nur a/make.tmpl.in b/make.tmpl.in
--- a/make.tmpl.in	2019-03-22 11:15:00.000000000 +0100
+++ b/make.tmpl.in	2020-03-26 15:18:39.800001043 +0100
@@ -91,7 +91,7 @@
 bindir = $(DESTDIR)@bindir@
 confdir = $(DESTDIR)@CONFDIR@/lvm
 profiledir = $(confdir)/@DEFAULT_PROFILE_SUBDIR@
-includedir = $(DESTDIR)@includedir@
+includedir = @includedir@
 libdir = $(DESTDIR)@libdir@
 libexecdir = $(DESTDIR)@libexecdir@
 usrlibdir = $(DESTDIR)@usrlibdir@
