diff -Nur a/Makefile.in b/Makefile.in
--- a/Makefile.in	2017-03-08 14:39:10.704047253 +0100
+++ b/Makefile.in	2017-03-08 14:56:44.632058557 +0100
@@ -21,9 +21,6 @@
 INSTALLED_CONFIGURATION_FILE=$(CONFIGURATION_FILE)
 INSTALLED_REVALIASES_FILE=$(REVALIASES_FILE)
 
-# Programs
-GEN_CONFIG=$(srcdir)/generate_config
-
 SRCS=ssmtp.c arpadate.c base64.c xgethostname.c @SRCS@
 
 OBJS=$(SRCS:.c=.o)
@@ -45,14 +42,13 @@
 	latex $<
 
 .PHONY: install
-install: ssmtp $(GEN_CONFIG)
+install: ssmtp
 	$(INSTALL) -d -m 755 $(bindir)
 	$(INSTALL) -s -m 755 ssmtp $(bindir)/ssmtp
 	$(INSTALL) -d -m 755 $(mandir)
 	$(INSTALL) -m 644 $(srcdir)/ssmtp.8 $(mandir)/ssmtp.8
 	$(INSTALL) -d -m 755 $(SSMTPCONFDIR)
 	$(INSTALL) -m 644 $(srcdir)/revaliases $(INSTALLED_REVALIASES_FILE)
-	$(GEN_CONFIG) $(INSTALLED_CONFIGURATION_FILE)
 
 
 .PHONY: install-sendmail
